docs/src/ref.md

```diff
@@ -6,4 +6,56 @@ CurrentModule = DispersiveShallowWater
 
 ```@autodocs
 Modules = [DispersiveShallowWater]
+Pages = ["DispersiveShallowWater.jl"]
+```
+
+## Equations
+
+```@autodocs
+Modules = [DispersiveShallowWater]
+Pages = [joinpath("equations", "equations.jl"), joinpath("equations", "bbm_bbm_1d.jl"), joinpath("equations", "bbm_bbm_variable_bathymetry_1d.jl"),
+         joinpath("equations", "svaerd_kalisch_1d.jl"), joinpath("equations", "serre_green_naghdi_flat_1d.jl")]
```

Comment:
I don't really like to list all files here. Do you know of any way to just include all files in `src/equations/` here automatically? I tried
```julia
Pages = readdir("equations")
```
but unfortunately that [doesn't seem to work](https://github.com/JoshuaLampert/DispersiveShallowWater.jl/actions/runs/10406337668/job/28819127781#step:7:13). [Here](https://documenter.juliadocs.org/stable/man/syntax/#@autodocs-block) is the documentation of `@autodocs` in Documenter.jl.

Reply:
Maybe you can insert a placeholder like `EQUATION_FILES_TO_BE_INSERTED` and substitute that with the files in a preprocessing step in `docs/make.jl`?

Reply:
Good idea. Let me try that.

Reply:
Let's see whether [aae0629](https://github.com/JoshuaLampert/DispersiveShallowWater.jl/pull/134/commits/aae0629e0ac645c8adbca5b6619b37f92400b331) works.

Reply:
Looks like it [worked](https://joshualampert.github.io/DispersiveShallowWater.jl/previews/PR134/ref/). Thanks for the hint!